announcements/1.17.1.md

```diff
@@ -0,0 +1,23 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.17.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* To minimise reboot of a machine or stopping of all containers, an internal check whether the GPU driver needs to be updated is in place. This also presents a status to the user indication the need to reboot the machine after a GPU driver update. The users can also ask if a pending upgrade will require a machine reboot or containers to be stopped. 
+* If the LXD charm is deployed on a machine that has an NVIDIA GPU installed, running the `sudo apt update && sudo apt upgrade -y` command for the machine may upgrade the NVIDIA drivers, which accidentally suspends running containers with GPU support. Hence, the NVIDIA drivers are held from being upgraded until you upgrade the LXD charm using the Juju command. You can check if the NVIDIA drivers are being held from being upgraded and when required, you can update them manually. 
+* Android security updates for March 2023 (see [Android Security Bulletin - March 2023](https://source.android.com/docs/security/bulletin/2023-03-01) for more information). Starting this month, Android 10 will no longer receive security updates as Google stops providing security patches for Android 10. 
+* The Android WebView has been updated to [111.0.5563.49](https://chromereleases.googleblog.com/2023/03/early-stable-update-for-android.html).
+
```

Comment:
We should add a warning for the incorrect security property. I will let @adglkh provide something.

Reply:
I'd suggest putting the following content to a 

## Know issues
Android 10 will no longer receive security patches from Google since this month, the security patch level for Android 10 should be `2023-02-05` rather than `2023-03-05`. This will be fixed in the Anbox Cloud 1.17.2 release.

Reply:
Sounds good to me.